ui/user/src/lib/components/Obot.svelte

```diff
@@ -96,14 +96,12 @@
 							{/if}
 						{/each}
 					{:else}
-						<div id="main-input" class="flex h-full max-w-full flex-1 justify-center">
```

Comment:
The main-innput ID was used here. Either that was already broken, it this will break. But the basic 
 idea was to focus the thread input when we switch or create threads from the sidebar. 
https://github.com/obot-platform/obot/blob/1d6764b1d1c6acf067c1d160cd728038574025db/ui/user/src/lib/components/sidebar/Threads.svelte#L81